Draft: Remove "simple", "simply", "easy", and "just" from the docs (#4496)
godotengine/godot-docs: tutorials/3d/lights_and_shadows.rst

.. image:: img/shadow_quadrants2.png

> The allocation logic consists of:
> 
> - The biggest shadow map size (when no subdivision is used), which represents a light the size of the screen (or bigger).
> - Subdivisions (smaller maps), which represent shadows for lights that are further away from view and proportionally smaller.

Reviewer: ```suggestion
The shadow atlas allocates space as follows:

- The biggest shadow map size (when no subdivision is used) represents a light the size of the screen (or bigger).
- Subdivisions (smaller maps) represent shadows for lights that are further away from view and proportionally smaller.
```
